packages/at_secondary_server/lib/src/verb/handler/abstract_update_verb_handler.dart

```diff
@@ -191,6 +191,12 @@ abstract class AbstractUpdateVerbHandler extends ChangeVerbHandler {
         verbParams[AtConstants.sharedKeyEncryptedEncryptingKeyName];
     metadata.skeEncAlgo =
         verbParams[AtConstants.sharedKeyEncryptedEncryptingAlgo];
+    if (verbParams[AtConstants.sharedWithPublicKeyHash] != null &&
+        verbParams[AtConstants.sharedWithPublicKeyHashingAlgo] != null) {
```

Comment:
In your changes to NotifyVerbHandler you are using `.isNotNullOrEmpty` - why the difference?

Reply:
@sitaram-kalluri can you respond to this comment please?